doc/user_manual/chExtraOutput.tex

```diff
@@ -65,15 +93,15 @@ \section{Dumping of Beam Population} \label{sec:DUMP}
         \hline
         Header & No header.\\
         \hline
-        Lines  & \texttt{turn structure\_element\_idx single\_element\_idx single\_element\_name s x1[m] x1'[rad] y1[m] y2'[rad] momentum[GeV/c] dE/E[GeV]} \\
+	    Lines  & \texttt{turn structure\_element\_idx single\_element\_idx single\_element\_name s x[m] xp[rad] y[m] yp[rad] P[GeV/c] (E-E0)[eV] t[s]}\\
         \hline
 
         \rowcolor{blue!15}
         \texttt{1} & Format for aperture check \\
         \hline
-        Header & \texttt{\# ID turn s[m] x[mm] xp[mrad] y[mm] yp[mrad] dE/E ktrack} \\
+	    Header & \texttt{\# ID turn s[m] x[mm] xp[mrad] y[mm] yp[mrad] dE/E ktrack} \\
```

Comment:
TAB character?

Reply:
just spaces for indentation. I am now going re-indent and split long lines.  I believe long cell in the table should have they own lines and tables should be indented...

Reply:
I would enforce 80 lines limit in latex source, since we concluded that for reading text 80 lines is more readable ;-)

Reply:
Please don't do that. I spent a lot of time reformatting the LaTeX source to conform to the standard @kyrsjo and others started to implement.
That is: a line break after each sentence.
Any decent editor has the option to wrap long lines, no? That solves the text readability.

Reply:
> wrapping line breaks indentation, for tables is a bit of a nightmare... but we can discuss...

Not really? My editor keeps indentation on wrapped lines, and so can vim as of about 4 years ago.

Reply:
Thanks for the hint! Very useful indeed. This does not solve the issue of the columns that happen at any random place in the line, but at least is more readable.